src/core_modules/capture-core/components/WidgetEnrollmentEventNew/AccessVerification/AccessVerification.container.js

```diff
@@ -12,9 +12,9 @@ import { makeEventAccessSelector } from './accessVerification.selectors';
 import type { ContainerProps } from './accessVerification.types';
 
 const dialogConfig = {
-    header: i18n.t('Unsaved changes'),
-    text: i18n.t('Leaving this page will discard the changes you made to this event.'),
-    confirmText: i18n.t('Yes, discard'),
+    header: i18n.t('Discard unsaved changes?'),
```

Comment:
The same suggestion here about reusing `header`, `text`, `destructiveText` and `cancelText` const values.